packages/application-extension/src/index.tsx

```diff
@@ -275,6 +275,11 @@ const tree: JupyterFrontEndPlugin<void> = {
         // Remove the tree portion of the URL leaving the rest intact.
         router.navigate(url);
 
+        const query = URLExt.queryStringToObject(args.search);
+        if (query.filebrowserPath) {
```

Comment:
I'd prefer to stick with URI tradition and use `file-browser-path` for the parameter name.   cf https://stackoverflow.com/a/18450653